src/components/reportPage/BatchJob.jsx

```diff
@@ -16,8 +19,11 @@ const BatchJob = () => {
     setEmail(event.target.value);
   };
 
-  const handleBatchJob = (files) => {
-    history.push("/batchsubmitpage", {email: emailRef.current});
+  const handleBatchJob = async (files, email) => {
```

Comment:
is it possible to omit the email as a param or is that bad coding practice? because if the email is omitted then you don't need an if statement in uploadSection.jsx line 61 to differentiate between live or batch job

Reply:
How this is done is a mess atm, this was the quickest way to get it working

Reply:
Fixed it